Source/Relativity.Test.Helpers/Relativity.Tests.Helpers.Tests.Unit/Kepler/OAuth2HelperUnitTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Moq;
+using NUnit.Framework;
+using Relativity.Test.Helpers.Kepler;
+using System.Threading.Tasks;
+
+namespace Relativity.Tests.Helpers.Tests.Unit.Kepler
+{
+	[TestFixture]
+	public class OAuth2HelperUnitTests
+	{
+		private Mock<IOAuth2Helper> Sut;

```

Comment:
you don't mock the actual Sut you are writing tests for. instead you mock the dependencies. can you fix it.

Reply:
Updated please review, also includes new RSAPI mock up helper